src/Operation/ListCollections.php

```diff
@@ -61,20 +61,27 @@ final class ListCollections implements Executable
      * @param array  $options      Command options
      * @throws InvalidArgumentException for parameter/option parsing errors
      */
-    public function __construct(private string $databaseName, array $options = [])
```

Comment:
I assume this was inadvertently added in a previous PR. Was this the only case?

Reply:
The database name was previously used in the `execute` method when constructing the iterator. This is no longer necessary, hence why we're no longer storing it as a class property.